### PR TITLE
Add Czech Subjectivity Classification Dataset

### PR DESCRIPTION
**Pull request by isaac-chung**
## Checklist for adding MMTEB dataset

Reason for dataset addition: Cover Czech in classification

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [POINTS.md](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) file. (depends on who reviews and merging of an approved PR)

### CODE REVIEW

**Review by imenelydiaker — APPROVED** [review]
Eveything is good, just added some suggestions! After addressing the comments, it should be good to merge!

File: mteb/tasks/Classification/ces/CzechSubjectivityClassification.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        reference="https://arxiv.org/abs/2009.08712",
+        dataset={
+            "path": "pauli31/czech-subjectivity-dataset",
+            "revision": "e387ddf167f3eba99936cff89909ed6264f17e1f",
+        },
+        type="Classification",
+        category="s2s",
+        date=("2022-04-01", "2022-04-01"),
+        eval_splits=["test"],
+        eval_langs=["ron-Latn"],
```
[the remark this answers is not in the document]
Comment (isaac-chung) [reply mteb/tasks/Classification/ces/CzechSubjectivityClassification.py:20]: 🙈 aaaah copy paste. my bad

**Comment by imenelydiaker** [comment]
Resolved the points.md conflicts, we're good for merging!

**Comment by isaac-chung** [comment]
@imenelydiaker thanks again! I'm curious, how did you push the `main` merge to this branch? I took a quite look and I wasn't sure. Did it involve setting a different remote?

**Comment by imenelydiaker** [comment]
Haha, actually I merged another PR to main while you were still working on this, and it resulted in a merge conflict because of point.md that was modified by the other PR. So I just used GitHub to fix it ☺️

**Comment by isaac-chung** [comment]
Do you mean the Github UI? I totally forgot that it's an option. Thanks! 🙌

**Comment by imenelydiaker** [comment]
Yes the UI! It's easier when it's just a markdown file to update, although I don't advise using it for fixing code conflicts, better do it locally and run the code before merging ☺️

**Comment by isaac-chung** [comment]
Most conflicts I've seen here so far involves `points.md`, so I guess the UI works. What steps would you take for code conflicts when you're trying to push to someone else's PR branch? I looked into [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork) but not sure I follow. Have you tried this before?

**Comment by imenelydiaker** [comment]
Generally, I solve it by merging the branch into mine (or doing a rebase, it almost the same thing), then resolving the conflicts manually on my editor (I don't think you can do better here, since conflicts are highlighted by a modification in the scripts (=== and >>>)). Finally, I push the changes to my branch after resolving (VSCode helps a lot with this btw). 

You can also use the GitHub UI to resolve conflicts, just make sure to run again the code before merging.

It's a bit hard to explain it without an example, and I'm not sure I have the best practices here niether 😅

**Comment by imenelydiaker** [comment]
> Most conflicts I've seen here so far involves `points.md`, so I guess the UI works. What steps would you take for code conflicts when you're trying to push to someone else's PR branch? I looked into [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork) but not sure I follow. Have you tried this before?

I think the link you sent is specific for forks. Although, a rebase or merging the main branch from the source repo into your fork's main branch should be good, also conflicts are generally resolved manually (just like I explained earlier)

**Comment by isaac-chung** [comment]
Gotcha. Resolving conflicts are common at work so no issues there, especially we would be working on the same repo, just different branches. I think the fork added a layer of confusion for me. Thanks for explaining :D

**Comment by imenelydiaker** [comment]
Just one thing, don't use GitHub UI to resolve conflicts on forks. We did it once and it changed the source branch of the fork, and we couldn't go back to the old branch. Anyway, it was painful 😅

**Pull request merged**